Source/JavaScriptCore/wasm/WasmCallee.cpp

```diff
@@ -421,7 +421,7 @@ RegisterAtOffsetList* JSEntrypointInterpreterCallee::calleeSaveRegistersImpl()
         RegisterSet registers = RegisterSetBuilder::wasmPinnedRegisters();
 #if CPU(X86_64)
 #elif CPU(ARM64) || CPU(RISCV64)
-        ASSERT(registers.numberOfSetRegisters() == 4);
+        ASSERT(registers.numberOfSetRegisters() == 3);
```

Comment:
Why is this 3 but JSEntrypointInterpreterCalleeSaveSpaceStackAligned is 6?

Reply:
This is the set of callee-save registers that are allowed to be trashed by the thunk. Every other callee save needs to be restored before calling into JS, in case we throw an exception. I'll find better names